nixos/modules/services/web-servers/darkhttpd.nix

```diff
@@ -0,0 +1,77 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.darkhttpd;
+
+  optionsToArgs = concatStringsSep " " ([
```

Comment:
A little bikeshedding on the naming: `optionsToArgs` would imply it taking an argument, but really it's just the value of the arguments, so I'd go for just `args` instead.